app/components/ranked-stats/ranked-stats-ctrl.js

```diff
@@ -31,19 +45,43 @@ angular.module('LeagueViewer')
       };
 
       var createStats = function() {
-        _($scope.champions).forEach(function(champion) {
-          champion.stats.winPercentage = champion.stats.totalSessionsWon / (champion.stats.totalSessionsPlayed);
+        ($scope.champions).forEach(function(champion) {
+          champion.stats.winPercentage = (champion.stats.totalSessionsWon / (champion.stats.totalSessionsPlayed)).toFixed(2);
```

Comment:
What do you think about saving win percentage as an actual percentage instead of decimal? Feel like its an easy improvement to make.